libraries/AP_Frsky_Telem/AP_Frsky_SPort_Passthrough.cpp

```diff
@@ -220,6 +230,23 @@ bool AP_Frsky_SPort_Passthrough::is_packet_ready(uint8_t idx, bool queue_empty)
 #endif
         }
         break;
+    case WIND:
+#if !APM_BUILD_TYPE(APM_BUILD_Rover)
+    {
+        float a;
+        WITH_SEMAPHORE(AP::ahrs().get_semaphore());
+        if (AP::ahrs().airspeed_estimate_true(a)) {
```

Comment:
I suspect this will end up being true on copters without a drag estimate (need drag estimate for wind)

Reply:
we probably need a have_wind_estimate() function